bookingticketui/src/components/ticket-form/ticket-form.jsx

```diff
@@ -0,0 +1,165 @@
+import React from "react";
+import { useState } from "react";
+import { createTicketAPIEndpoint, ENDPOINTS } from "../../api/api-collector";
+import { useNavigate } from "react-router-dom";
+import { displayAlert } from "../notification/toast";
+import Card from "react-bootstrap/Card";
+import ListGroup from "react-bootstrap/ListGroup";
+import { format } from "date-fns";
+import "react-toastify/dist/ReactToastify.css";
+import "./ticket-form.css";
+
+const DATE_OPTIONS = {
+  weekday: "short",
+  year: "numeric",
+  month: "short",
+  day: "numeric",
+};
+
+const Ticket = () => {
+  const [data, setData] = useState({});
+  const [tickets, setTickets] = useState([]);
+  const [formStatus, setFormStatus] = useState("Check");
+  const [show, setShow] = useState(false);
+  const navigate = useNavigate();
+  const [conForm, setConFom] = useState({
+    Code: "",
+  });
+  const handleChange = (event) => {
+    setConFom({
+      ...conForm,
+      [event.target.name]: event.target.value,
+    });
+  };
+
+  const submitForm = async (event) => {
+    event.preventDefault();
+    console.log(conForm.Code);
+    let result = await createTicketAPIEndpoint(ENDPOINTS.bill, conForm.Code);
+
+    let message = "";
+    let type = "";
+
+    if (result?.response?.status == 500) {
+      console.log(result?.response?.status);
+      message = "The code does not exist";
+      type = "error";
+      displayAlert(message, type);
+      setShow(false);
+    } else {
+      message = "Get succefully";
+      type = "success";
+      displayAlert(message, type);
+      setData(result);
+      setTickets(result.result?.data?.tickets);
+      console.log(result.result?.data?.tickets);
+      console.log(result);
+      setShow(true);
+    }
+  };
+
+  return (
+    <div>
+      <div className="ticket-container">
+        <h1 id="contactheader">Ticket detail</h1>
+
+        <form onSubmit={submitForm}>
+          {/* Reference cdoe form   */}
+          <div className="input-box">
+            <label className="form-label" htmlFor="Code"></label>
+            <input
+              style={{
+                borderRadius: "5px",
+                border: "1px solid",
+              }}
+              className="form-control"
+              type="text"
+              name="Code"
+              id="Code"
+              onChange={handleChange}
+              required
+            />
+          </div>
+
+          <button className="action-button" type="submit">
+            Check!
+          </button>
+
+          <div className="information">
+            {show ? (
+              <div
+                style={{
+                  display: "flex",
+                  gap: "10px",
+                  flexWrap: "wrap",
+                  justifyContent: "center",
+                }}
+              >
+                {tickets.map((getcon) => (
```

Comment:
getcon??

Reply:
fixed. Changed the name to "items" to make more sense